sys-auth/polkit/polkit-122-r1.ebuild

```
@@ -149,6 +149,7 @@ src_install() {
 			diropts -m 0700 -o polkitd
 		fi
 		keepdir /etc/polkit-1/rules.d
+		install -D -m 0644 ${FILESDIR}/override.conf /etc/systemd/system/polkit.service.d/override.conf
```

This looks like it won't respect `DESTDIR` and hence trigger a sandbox violation, so:
```suggestion
		dodir /etc/systemd/system/polkit.service.d
		install -D -m 0644 "${FILESDIR}"/override.conf ${DESTDIR}/etc/systemd/system/polkit.service.d/override.conf || die
```

but even better:
```suggestion
		insinto /etc/systemd/system/polkit.service.d
		doins "${FILESDIR}"/override.conf
```

But see my other comment first.